### ISSUE TITLE
Unable to run "Sentinel 2 - AWS COG" example

### ISSUE THREAD

**Issue by aisacode** [cogeotiff/rio-tiler-pds#49]
I'm trying to run the following snippets, as reported in the [documentation ](https://cogeotiff.github.io/rio-tiler-pds/usage/sentinel/#cog-only-l2a-available-for-now)

```
from rio_tiler_pds.sentinel.aws import S2COGReader

with S2COGReader("S2A_29RKH_20200219_0_L2A") as sentinel:
    print(type(sentinel))
```

I got the following error:

`**NoCredentialsError: Unable to locate credentials**`

In my understanding, sentinel2 COGs are accessible without any AWS credentials.

Moreover, [recently](https://www.element84.com/blog/sentinel-2-cloud-optimized-geotiffs-now-available-on-aws-registry-of-open-data), global sentinel 2 COGs was made publicly available. There is some ways to access them trough rio_tiler_pds? If yes, documentation still refer only to Africa related COG.

**Comment by kylebarron** [comment]
By default `boto3`, the underlying library for accessing data on AWS, tries to access AWS credentials even for public data. Hence the easiest way forward would be for you to create AWS credentials locally.

Alternatively it should be possible to create an anonymous boto3 session where requests are unsigned. I'd have to look closer at the code to figure out how to do that here.

**Comment by kylebarron** [comment]
> There is some ways to access them trough rio_tiler_pds? If yes, documentation still refer only to Africa related COG.

Yes you can access any Sentinel 2 scene via its scene id. The docs should probably be updated to reflect that Sentinel 2 is available globally. (I thought they already were but maybe I'm mistaken)

**Comment by kylebarron** [comment]
Alternatively we could provide an option to the `S2COGReader` to load files over HTTP directly.

**Comment by aisacode** [comment]
> By default `boto3`, the underlying library for accessing data on AWS, tries to access AWS credentials even for public data. Hence the easiest way forward would be for you to create AWS credentials locally.
> 

Thanks, create local credentials works!
Anyway, having a different default setting (no credentials needed) would be very nice for public data
